### ISSUE TITLE
About the training data resolution

### ISSUE THREAD

**Issue by austingg** [DeepScale/SqueezeNet#8]
Hi, 
I have a question about the training data resolution of SqueezeNet. Is it resized to 256x256 or with the smaller side 256 px. I have use both resolutions to test your pretrained model. Seem 256x256 resolution get a higher accuracy.

**Comment by forresti** [comment]
I believe our training data is stretched so that is square 256x256. There
are certainly alternatives to this that will lead to higher accuracy
(though if you change the input dimensions a lot, you may want to retrain).

**Comment by austingg** [comment]
@forresti got it, thanks.
